src/backend/functions/work-packages-create.ts

```diff
@@ -38,16 +24,22 @@ export const createWorkPackage: Handler = async ({ body }, _context) => {
       goals: true,
       features: true,
       otherConstraints: true,
```

Comment:
Not sure why these other properties are being asked for, I know you didn't touch them, but I believe these 3 can be removed without any effect.

Reply:
true, good point